### DISCUSSION TITLE
Can you please reply my email 🙏

### DISCUSSION THREAD

**Discussion by muhdazrain** [vercel/community#1039] [Help] [answered]
Hi,
I just didn't know where to put but I really need Vercel Support to reply my email as fast as possible because currently I cannot deploy any bug fixes to my production.

So I hope Vercel can help to resolve this.

![Screenshot_20221109_222520_com zoho mail](https://user-images.githubusercontent.com/1797935/200861048-2c116418-d226-4237-834e-4d8c68972e32.jpg)

Thanks

**Comment by amyegan** [comment, marked as answer]
Hi @muhdazrain! Do you know the case number? That will let me find the case details and get you the help you need 🙏

**Reply by muhdazrain** [reply]
> Hi @muhdazrain! Do you know the case number? That will let me find the case details and get you the help you need 🙏

Hi @amyegan ,
You may refer either these two tickets. Not sure which one latest.

- Case #00106082
- Case #00106214

Thank you very much.

**Reply by amyegan** [reply]
Thanks for sharing these. Looking through the logs, I'm seeing different versions of Vercel CLI being used for the builds. Can you try setting an [environment variable](https://vercel.com/docs/concepts/projects/environment-variables) named `VERCEL_CLI_VERSION` with a value like `vercel@28.4.11` and then try to deploy again. That's a temporary way for us to check if a CLI update is causing this build error. If that solves your problem, then we will have a better idea of where to look for a more permanent solution.

I'm sorry I cannot try this for you. For safety and security reasons, I do not have access to make account settings updates or deployments on your behalf.

**Reply by muhdazrain** [reply]
@amyegan I have tried put in the environment variables but still same error.

**Reply by amyegan** [reply]
It looks like you have been working with the success team on those two tickets and are making some progress with them, so I'm going to let you continue the conversation there. They have more details at this point so I think they will be best able to help you solve this problem.
